src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/SyntaxVisualizerToolWindow.cs

```diff
@@ -31,6 +31,7 @@ internal class SyntaxVisualizerToolWindow : ToolWindowPane
     private const int CmdIdShowAllTagHelpers = 0x0113;
     private const int CmdIdShowInScopeTagHelpers = 0x0114;
     private const int CmdIdShowReferencedTagHelpers = 0x0115;
+    private const int CmdidShowFormattingDocument = 0x0117;
```

Comment:
>7 [](http://example.com/codeflow?start=57&length=1)

Nit: intentionally skipped 0x0116?

Reply:
Oh, lol, I used that is to try to get a dropdown working, but the vsct magic eluded me. I will update :)

Reply:
There's no magic! VS is just GUIDs and ints all the way down. 😁